CaseStudies/ssp/docs/SRS/SRS.tex

```diff
@@ -128,23 +128,25 @@ \subsection{Table of Units}
 \setlength{\tabcolsep}{20pt}
 \begin{tabular}{  l  l  l  }
 \hline
-\textbf{Physical Property} & \textbf{Name} & \textbf{Symbol} \\
+\textbf{Symbol} & \textbf{Unit} & \textbf{SI} \\
 \hline
-force & Newton & \si{\newton} \\
-length & meter & \si{\meter}  \\
-pressure & Pascal & $\si{\pascal}=\si{\newton\per\square\meter}$ \\
-angle & degree & \si{\degree}  \\
+\si{\newton} & force & Newton \\
+\si{\meter} & length & meter \\
+$\si{\pascal}=\si{\newton\per\square\meter}$ & pressure & Pascal \\
```

Comment:
Pascal should be pascal.  All units named after people are actually not capitalized.  This is a good example for @szymczdm in his writing.  We fixed this mistake quickly in all of the Drasil examples, but the manual examples still have the incorrect capitalization after all this time.